.dockerignore

```diff
@@ -0,0 +1,4 @@
+.github
+.gitignore
+appveyor.yml
+azure-pipelines.yml
```

Comment:
Those files no longer exist

Reply:
oh right thanks. resolving that before merging.